app/controllers/photos_controller.rb

```diff
@@ -15,13 +15,24 @@ def multi_delete
     end
 
     if params['photo']['id'].present?
-      @album.photos.where("id IN (?)",params[:photos][:id]).destroy_all
+      @album.photos.where("id IN (?)",params[:photo][:id]).destroy_all
     else
       @album.photos.destroy_all
     end
     json_response({success: true, message: "Selected photos deleted successfully."}, 200)
   end
 
+  def set_cover_photo
+    # binding.pry
+    @photo = @album.photos.find(params[:id])
+    if @photo
+      @photo.set_as_cover
+      json_response({success: true, message: "Set as cover photo successfully."}, 200)
+    else
+      json_response({success: false, message: "photo not found"}, 400)
+    end
```

Comment:
@arps1989dev  Please Remove else condition from here, because this will handle by exception_handler method
And in success response please add `@photo` in success response.